### PR TITLE
Set up a 'Hello Microverse' project Pull request

### PR DESCRIPTION
**Pull request by Fentahunn**
First hello micro verse project to understand the concepts of linter.

- [ ]  I have added the requested files like readme and test.md files. 
- [ ] @Zuhaib042 (I have completed the task that you requested me, so please review my work ?)

- [ ] @shella12 (I have changed the requested readme file. So am requesting to get a review of my work. )

### CODE REVIEW

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @Fentahunn , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

## Highlights
✅ No linters errors

## Required Changes ♻️
- [ ] Kindly consider adding a readme file to your feature branch. you can get the readme from the following template and you must customize it and fill all the sections in with relevant information. you may also delete the Microverse logo. 
  -  Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
- [ ] Kindly consider adding a test.md file with this text `This file should be ignored by git` to this feature branch and then also add a `.gitignore` file to this feature branch. In that .gitignore file, you may add `test.md` and `node_modules` on separate lines.
   -  A test.md file with the text "This file should be ignored by git".
- [ ] The first word of your commit message should be capitalized so kindly consider taking care of this in your future projects.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,283 +1,281 @@
...
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
```
Comment (shella12) [inline README.md]: - [ ] Great job using READMe Template however please make sure to customise each section with the project description that reflects  your project and delete irrelevant things for example databases, REACT or,  express.js form your README and include tech stack that you have applied to your project e.g HTML, CSS etc

File: README.md [hunk 2]
```diff
@@ -1,283 +1,281 @@
...
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The project consits of
```
Comment (shella12) [inline README.md]: - [ ] - Please remove the default comment like 
> Add a link to your deployed project.

> Describe how a new developer could make use of your project

>    Describe 1 - 3 features you will add to the project.

  ,etc from your README. They were used in the template to explain to the user the purpose of each section. You can go ahead and remove it from your README

File: README.md [hunk 3]
```diff
@@ -1,283 +1,281 @@
...
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
```
Comment (shella12) [inline README.md]: - [ ] FAQ is an OPTIONAL section if you don't fell like adding anything to it  please remove it from the README.

File: README.md [hunk 4]
```diff
@@ -1,283 +1,281 @@
...
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (shella12) [inline README.md]: - [ ] The link to your `MIT LICENCE` is broken. Please use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly add LICENCE into your README.

File: README.md [hunk 5]
```diff
@@ -1,283 +1,281 @@
...
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (shella12) [inline README.md]: - [ ] Please make sure to customise `Author` and `Future features `section to reflect your project

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @Fentahunn  

Great work on making the changes requested by a previous reviewer 👏 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. You are almost there...



## Highlights

- All linters passed ✅ 
- Used README.md Template✅   

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 6]
```diff
@@ -1,174 +1,174 @@
...
+
+I would like to thank my coding prartner,.They are really supportive.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+- **How it has been done**
+
+- **what are the key features**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (shella12) [inline README.md:166]: - [ ] [OPTIONAL] `FAQ` ia an `OPTIONAL` part of `README` however if you have decided to include it in the `README` I would suggest you put the `FAQ heading` here as well ` (## ❓FAQ)`

**Review by shella12 — APPROVED** [review]
Hi @Fentahunn , 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Linters are passing ✔️ 
- README is professional 👍 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**